drivers/disp_dev/disp_dev.c

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (C) 2020 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @{
+ *
```

Comment:
doesn't this need an `@ingroup`?

Reply:
Thanks, that makes me realized that the module group is invalid now. I'll fix.